README.md

```diff
@@ -12,7 +12,7 @@ Este proyecto fue desarrollado como Proyecto Final para el curso de Web Developm
 
 ## Para ejecutar
```

Comment:
**Good job!** 
- Adding information on how to run your project in a local environment. 👏🏽🤩